testutil/integration/evmos/network/network.go

```diff
@@ -108,9 +108,7 @@ var (
 func (n *IntegrationNetwork) configureAndInitChain() error {
 	// Create funded accounts based on the config and
 	// create genesis accounts
-	coin := sdktypes.NewCoin(n.cfg.denom, PrefundedAccountInitialBalance)
-	genAccounts := createGenesisAccounts(n.cfg.preFundedAccounts)
-	fundedAccountBalances := createBalances(n.cfg.preFundedAccounts, coin)
```

Comment:
Should we delete this unused functions from the network package now?

Reply:
yeah good catch 👍

Reply:
ah actually they are still used 😅 so I'll keep them around, in fact I've reused them in the refactor mentioned above